matrix/rooms.py

```diff
@@ -42,7 +42,8 @@ def __init__(self, room_id):
         # type: (str) -> None
         # yapf: disable
         self.room_id = room_id        # type: str
-        self.alias = room_id          # type: str
```

Comment:
This is used in matrix_bar_item_name() in bar_items.py to get the buffer name in the status bar.